### ISSUE TITLE
Parallel project : Cartes.app

### ISSUE THREAD

**Issue by laem** [openstreetmap-ng/openstreetmap-ng#124]
Hi, you might be interested by our project : https://cartes.app, fully open at github.com/laem/cartes. 

It doesn't have the ambition to replace the osm stack, far from it, just the end-user Web UI, and make a connection to all the existing relevant databases that let us rival with Google Maps. Given the challenge, it's France-only for the first stage. 

It won't replace osm.org either, it's a different project, consuming and combining multiples dbs and APIs (international and local), not just OSM. 

It has several innovative features compared to the current state of Web OSM UIs, I'll let you try to see if it looks interesting to you, french language aside :p 

In particular, we're trying to tackle the problem of transit. We're spending 50 % of our time on the transit UI and backend, aiming at multimodal trips on par with Google. We're building with ecology in mind (not as a car-centric burn the planet perspective). 

Is there a document somewhere were the role of openstreetmap-ng is discussed, in terms of integration to the osm community ? 

Thanks for the hard open source work :)

**Comment by Zaczero** [comment]
Hey, Is it possible for you to join our discord (see readme)? This is where most of the discussion happens. We can schedule a call/event. OSM is in the strong need of UI/UX improvement and every helping hand helps. Any ideas are also welcome too!

**Comment by laem** [comment]
Hi ! To be frank, I don't think I'll have time to participate in the discord discussions (if only it were Matrix, a free alternative !) nor wanting to create a new account on this platform and follow notifications there. I'm already struggling with github projects, mail, OSM's forums, and matrix groups, and 100 % of my free time on Cartes :sweat_smile: 

Considering the ambitions of both projects, I'm quite favorable to the principle of keeping the separation between back+contributor UI on one side, and general public interface on the other side, but staying informed about their new features.

But  I wanted to know how far did you go in terms of potential replacement of osm.org. Is it discussed somewhere, on a public page, rather than instant messaging ? Is there some kind of alignment with the OSM community ? Will both coexist for some time ? Are they contributing to the same DB ? What's the plan ?

**Comment by kudlav** [comment]
Another parallel project for end user web UI is [osmapp.org](https://osmapp.org/)

**Comment by laem** [comment]
Yes, good point, we've talked a bit, we're using the same techs as osmapp, son sharing new features is quite easy. Cartes.app wants to be a more complete local (France) alternative to Gmap, osmapp is an international OSM frontend. I believe both are needed right now.

**Comment by natrius** [comment]
I have created a space on matrix ( https://matrix.to/#/#osm-ng:matrix.org ) with two rooms, general and support, i would be happy to bridge these two channels to the two discord-channels. I know quite a lot of OSM-People are on matrix, so it might get some people as well. 

If you would like that, we should talk on discord and then create the bridge, you can read up there https://t2bot.io/discord/ the short descriptions.

**Comment by laem** [comment]
Thanks ! Makes me think of https://areweturboyet.com

Cool way to showcase work being done and reassure about the state of a large scale rewriting. 

![image](https://github.com/user-attachments/assets/91fbef74-c613-4637-a531-79883d44c025)
